packages/plugin_platform_interface/README.md

```diff
@@ -35,14 +35,15 @@ abstract class UrlLauncherPlatform extends PlatformInterface {
 This guarantees that UrlLauncherPlatform.instance cannot be set to an object that `implements`
 UrlLauncherPlatform (it can only be set to an object that `extends` UrlLauncherPlatform).
 
-## Mocking platform interfaces with Mockito
+## Mocking or faking platform interfaces
```

Comment:
You might want to mention this in the CHANGELOG as well.

Reply:
Makes sense, done.